vagrant/Makefile

```diff
@@ -18,7 +18,7 @@ setup.test.only:
 	@ansible-playbook --inventory=$(INVENTORY) setup.test.yml
 
 setup.cluster.only:
-	@ansible-playbook --inventory=$(INVENTORY) setup.run.yml
+	@vagrant ssh -c "ansible-playbook -i /home/vagrant/ansible/vagrant_ansible_inventory /home/vagrant/ansible/setup-cluster.yml" setup
```

Comment:
You could just do `make setup.cluster` on the remote machine?

Reply:
ohh sorry. Just saw this. I'll check that  out.